gcp/modules/gcp-stackdriver-monitoring/resources/alert_policies/backup_exporter_errors.json

```diff
@@ -0,0 +1,38 @@
+{
+  "display_name": "Error Check for Backup-exporter",
```

Comment:
Please update this to match the new, action-oriented naming scheme that Sergey merged recently.

Reply:
Also the other check in the other file.